lhotse/supervision.py

```diff
@@ -512,6 +512,50 @@ def from_dicts(data: Iterable[Dict]) -> "SupervisionSet":
             SupervisionSegment.from_dict(s) for s in data
         )
 
+    @staticmethod
+    def from_rttm(path: Pathlike) -> "SupervisionSet":
+        """
+        Read an RTTM file located at ``path`` and create a :class:`.SupervisionSet` manifest for them.
+        Can be used to create supervisions from custom RTTM files (see, for example, :class:`lhotse.dataset.DiarizationDataset`).
+
+        Use the :meth:`.from_rttms` method if you want to read multiple RTTM files at once.
```

Comment:
How about modifying this function to accept both a single Pathlike and an iterable of them (you can check with `isinstance`)?